recipes/easyhttpcpp/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_subdirectory("source_subfolder")
```

```suggestion
conan_basic_setup()

if(NOT CMAKE_CXX_STANDARD)
    set(CMAKE_CXX_STANDARD 11)
endif()

add_subdirectory("source_subfolder")
```
@uilianries  Would this be *correct*?

---

Yes, few other recipes follow this solution.

---

Thanks